docs/setup.md

```diff
@@ -120,69 +121,66 @@ The Settings Page is available from the **File** menu under **Settings** (Click
+If **Hide Menu Bar** option is enabled, use the `ALT` key to toggle the menu on and off.
 
-Below lists menu options (shortcut keys are listed in brackets, `Ctrl` becomes `Command` on OS X):
+Menu options are listed below (shortcut keys are listed in brackets, `CTRL` becomes `CMD` on macOS):
 
 - **File**
-  - **About Mattermost** - Shows version information for Mattermost desktop application
-  - **Settings** (Ctrl+Comma) - Opens setting menu to add new team sites and configure shortcut key options
-  - **Quit** (Ctrl+Q) - Exits the application
+  - **About Mattermost** - Show version information for Mattermost desktop application
+  - **Settings** (CTRL+,) - Open setting menu to add new team sites and configure shortcut key options
+  - **Quit** (CTRL+Q) - Exit the application
 - **Edit**
-  - **Undo** (Ctrl+Z) - Reverses previous action
-  - **Redo** (Ctrl+Shift+Z, Ctrl+Y on Windows) - Replays most recent action
-  - **Cut** (Ctrl+X) - Cuts selected text
-  - **Copy** (Ctrl+C) - Copies selected text
-  - **Paste** (Ctrl+V) - Pastes text from clipboard
-  - **Select All** (Ctrl+A) - Select all text in input box
-  - **Search in Team** (Ctrl+S) - Puts cursor in search box to search in the current team
+  - **Undo** (CTRL+Z) - Reverse previous action
+  - **Redo** (CTRL+SHIFT+Z, CTRL+Y on Windows) - Replay most recent action
+  - **Cut** (CTRL+X) - Cut selected text
+  - **Copy** (CTRL+C) - Copy selected text
+  - **Paste** (CTRL+V) - Paste text from clipboard
+  - **Select All** (CTRL+A) - Select all text in input box
+  - **Search in Team** (CTRL+S) - Put cursor in search box to search in the current team
 - **View**
-  - **Find..** (Ctrl+F)- Find in page
-  - **Reload** (Ctrl+R) - Reload page from the server
-  - **Clear Cache and Reload** (Ctrl+Shift+R) - Clear cached content in application and reload page
+  - **Find..** (CTRL+F)- Find in page
+  - **Reload** (CTRL+R) - Reload page from the server
+  - **Clear Cache and Reload** (CTRL+SHIFT+R) - Clear cached content in application and reload page
   - **Toggle Full Screen** (F11) - Toggle application from window to full screen and back
-  - **Toggle Developer Tools** (Ctrl+Shift+I) - Turn on and off sidebar showing developer tools
+  - **Toggle Developer Tools** (CTRL+SHIFT+I) - Turn on and off sidebar showing developer tools
   - **Actual Size** (Ctrl+0) - Reset zoom level
-  - **Zoom In** (Ctrl+Plus) - Enlarge the rendered contents size
-  - **Zoom In** (Ctrl+Minus) - Shrink the rendered contents size
+  - **Zoom In** (CTRL++) - Enlarge the rendered contents size
+  - **Zoom In** (CTRL+-) - Shrink the rendered contents size
 - **History**
-  - **Back** (Alt+Left, Command+[ on OS X) - Go back to previous web page in the current tab
-  - **Forward** (Alt+Right, Command+] on OS X) - Go forward to next web page in the current tab
+  - **Back** (ALT+LEFT, CMD+`[` on macOS) - Go back to previous web page in the current tab
```

Comment:
Please could you confirm this command as it looks similar to the one below. Thank you!

Reply:
ALT+LEFT and ALT+RIGHT work on Linux, got no Mac to test this on

Reply:
Thanks for testing! I tested on macOS and can confirm the listed command works. I just changed the formatting. :)